### ISSUE TITLE
flynn-cli for Linux x86_64 crashes.

### ISSUE THREAD

**Issue by thomasfedb** [flynn/flynn#1127]
Hey, I've just installed flynn-cli as per instructions in `cli/README.md` and encountered the following issue when I attempted to run `flynn`:

```
# flynn
panic: user: Current not implemented on linux/amd64

goroutine 1 [running]:
github.com/flynn/flynn/installer.init·1()
	/Users/titanous/src/gopkgs/src/github.com/flynn/flynn/installer/persistance.go:21 +0x6b
github.com/flynn/flynn/installer.init()
	/Users/titanous/src/gopkgs/src/github.com/flynn/flynn/installer/stack_template.go:237 +0x203
main.init()
	/private/tmp/flynn-build/cli/version.go:17 +0xb8

goroutine 5 [syscall]:
os/signal.loop()
	/Users/titanous/src/go-tip/src/os/signal/signal_unix.go:21 +0x1f
created by os/signal.init·1
	/Users/titanous/src/go-tip/src/os/signal/signal_unix.go:27 +0x35

goroutine 6 [chan receive]:
github.com/flynn/flynn/pkg/shutdown.(*handler).wait(0xc20803aa50)
	/Users/titanous/src/gopkgs/src/github.com/flynn/flynn/pkg/shutdown/handler.go:54 +0x158
created by github.com/flynn/flynn/pkg/shutdown.newHandler
	/Users/titanous/src/gopkgs/src/github.com/flynn/flynn/pkg/shutdown/handler.go:25 +0xa8

goroutine 8 [runnable]:
text/template/parse.lexText(0xc20807c580, 0xb08ee0)
	/Users/titanous/src/go-tip/src/text/template/parse/lex.go:228 +0x37b
text/template/parse.(*lexer).run(0xc20807c580)
	/Users/titanous/src/go-tip/src/text/template/parse/lex.go:198 +0x5d
created by text/template/parse.lex
	/Users/titanous/src/go-tip/src/text/template/parse/lex.go:191 +0x1ac
```

It appears that this may be related to crosscompilation, see [this Go issue](https://code.google.com/p/go/issues/detail?id=6376#c2).

**Comment by thomasfedb** [comment]
Conversation with @lmars resulted in a temporary solution, using [a different flynn-cli binary](https://s3.amazonaws.com/flynn/tuf/targets/4b2fea62b5c07de733037cf5bddd55f9c166acaae3e564538db569f782aee937387636b164acec48d55110dd3dc1d0c605b254b053487971069a32684c46941f.flynn-linux-amd64.gz) for my system. Issue will likely be resolved as part of #1087.

**Comment by titanous** [comment]
https://github.com/inconshreveable/gonative is a good solution for this.

**Comment by titanous** [comment]
I've released `v20150302.2` which seems to work fine on Linux.
